### DISCUSSION TITLE
Bungie.net Destiny API error

### DISCUSSION THREAD

**Discussion by FireFighter1017** [grafana/grafana-infinity-datasource#1084] [Specific APIs]
When I try to connect to Bungie's API with my developers API key, it fails with this error: 
`Error: Get "https://www.bungie.net/Platform/Destiny2/Manifest/": oauth2: "unsupported_grant_type" "Only authorization_code or refresh_token supported."`

Here's my config: 
![image](https://github.com/user-attachments/assets/3c455350-ca17-410f-b284-ee65edde85c1)

**Comment by yesoreyeram** [comment]
As documented and seen in the UI, only client credentials and jwt type oauth2 is supported in the plugin for now
